README.md

```diff
@@ -117,6 +117,21 @@ This section contains frequently-asked questions regarding working with Finch.
 LIMA_HOME=/Applications/Finch/lima/data /Applications/Finch/lima/bin/limactl shell finch
 ```
 
+#### How to authenticate to GCR(Google Container Registry) or AR(Artifact Registry)?
+Create a [GCP Service Account](https://cloud.google.com/iam/docs/service-accounts-create#creating), grant `Owner` role, and download the key as a JSON file to the $HOME directory.
+
+Then shell into the VM by running the following command:
+```sh
+LIMA_HOME=/Applications/Finch/lima/data /Applications/Finch/lima/bin/limactl shell finch
```

Comment:
I think we should not put `vm shell` as part of any user's workflow, unless it is a very temporary workaround. If we want to add this to doc, let's highlight it is workaround and link an issue for the right solution here.

Reply:
Added link to the issue and specify it's a temp workaround way.